static/app/views/onboarding/otherSetup.tsx

```diff
@@ -0,0 +1,179 @@
+import 'prism-sentry/index.css';
+
+import * as React from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import {openInviteMembersModal} from 'app/actionCreators/modal';
+import {Client} from 'app/api';
+import Alert from 'app/components/alert';
+import AsyncComponent from 'app/components/asyncComponent';
+import Button from 'app/components/button';
+import ExternalLink from 'app/components/links/externalLink';
+import LoadingError from 'app/components/loadingError';
+import {PlatformKey} from 'app/data/platformCategories';
+import platforms from 'app/data/platforms';
+import {t, tct} from 'app/locale';
+import {Organization, Project} from 'app/types';
+import {analytics} from 'app/utils/analytics';
+import getDynamicText from 'app/utils/getDynamicText';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import {ProjectKey} from 'app/views/settings/project/projectKeys/types';
+
+import FirstEventFooter from './components/firstEventFooter';
+import SetupIntroduction from './components/setupIntroduction';
+import {StepProps} from './types';
+
+type AnalyticsOpts = {
+  organization: Organization;
+  project: Project | null;
+  platform: PlatformKey | null;
+};
+
+const recordAnalyticsDocsClicked = ({organization, project, platform}: AnalyticsOpts) =>
+  analytics('onboarding_v2.full_docs_clicked', {
+    org_id: organization.id,
+    project: project?.slug,
+    platform,
+  });
+
+type Props = StepProps & {
+  api: Client;
+  organization: Organization;
+} & AsyncComponent['props'];
+
+type State = {
+  platformDocs: {html: string; link: string} | null;
+  loadedPlatform: PlatformKey | null;
+  hasError: boolean;
+  keyList: Array<ProjectKey> | null;
+} & AsyncComponent['state'];
+
+class OtherSetup extends AsyncComponent<Props, State> {
+  getDefaultState() {
+    return {
+      ...super.getDefaultState(),
+      platformDocs: null,
+      loadedPlatform: null,
+      hasError: false,
+      keyList: null,
+    };
+  }
+
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, project} = this.props;
+    return [['keyList', `/projects/${organization.slug}/${project?.slug}/keys/`]];
+  }
+
+  handleFullDocsClick = () => {
+    const {organization, project, platform} = this.props;
+    recordAnalyticsDocsClicked({organization, project, platform});
+  };
+
+  render() {
+    const {organization, project, platform} = this.props;
+    const {loadedPlatform, hasError, keyList} = this.state;
+
+    const currentPlatform = loadedPlatform ?? platform ?? 'other';
+
+    const introduction = (
+      <React.Fragment>
+        <SetupIntroduction
+          stepHeaderText={t(
+            'Prepare the %s SDK',
+            platforms.find(p => p.id === currentPlatform)?.name ?? ''
```

Comment:
@roggenkemper since we know this is the `other` platform, we can hard code the name

Reply:
done!